yarn.lock

```diff
@@ -2721,9 +2721,8 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001181:
-  version "1.0.30001183"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001183.tgz#7a57ba9d6584119bb5f2bc76d3cc47ba9356b3e2"
-  integrity sha512-7JkwTEE1hlRKETbCFd8HDZeLiQIUcl8rC6JgNjvHCNaxOeNmQ9V4LvQXRUsKIV2CC73qKxljwVhToaA3kLRqTw==
+  version "1.0.30001248"
+  resolved "https://registry.nlark.com/caniuse-lite/download/caniuse-lite-1.0.30001248.tgz"
```

Comment:
Hi, I just happened to see the PR and was checking the diff.

This line shows that the package download location has been rewritten, is this what you intended? Downloading a tar from an unknown domain looks very dangerous...

Reply:
It's changed because I use a mirror registry for installation, It's trusted , I've revert to previous version , dont know this PR could be merged or not , if it's acceptable I'll change package name back.

Reply:
Thanks for the reply!  I also appreciate the quick fix.

>It's changed because I use a mirror registry for installation

If you don't mind me asking, could you tell me why you used the mirror registry? Is it due to geographical reasons? From my point of view, the npm registry seems sufficient, but from your point of view, I expect that the npm registry may not be sufficient.

>It's trusted

I've never seen registry.nlark.com before, so for a moment I couldn't decide if you were right or not. At least in Japan, where I live, this registry is not trusted. But from what I could find out, registry.nlark.com seems to be a registry used by many Chinese users for sure!

I guess people will differ on whether it is trustworthy or not, but I will leave the decision to the browserslist-useragent-regexp's maintainers.

Reply:
In China we are behind GFW, so they have network condition, we almost use mirror resource everywhere including npm, linux package, docker image. they may have local user configuration so the lock file may including packages from other registry.  registry.nlark.com is maintained by Alibaba.

Reply:
Oh, I understood the situation just now. That sounds tough...